### PR TITLE
all snack break pages reformatted + act 5 friendquests reformatted + BONNIEQUEST!

### PR DESCRIPTION
**Pull request by candycoatedrox**
bonniequest page also *exists* but doesn't have anything on it aside from copied dialogue from loop + snack break pages! working on it!

### CODE REVIEW

**Comment by goldstargloww** [comment]
converting to draft until the bonniequest page is done :D

**Comment by goldstargloww** [comment]
as for what's here currently:
- bonniequest
  1. if i'm reading the (...) bits correct, they're meant to be like, "dialogue goes here but i'm omitting it", right? in that case, [...] is often used and i think it would read clearer - even so it's up to feli if we do that at all
- isaquest (i realized after writing this that you literally only changed 2 characters in isaquest. but i've written this much and don't wanna delete it anymore. oops)
  1. re line 585, "Before the quest & ACT 3", act 4 doesn't start until all friendquests have been done so this can just say act 3
  2. re lines 740-756, "So, what do you need me to do?", it's not specified what expression happens when
  3. re lines 780-796, "First time: (...?) (You lay down.)" / "Subsequent times: (You lay down.)" - even if it isn't in the code, this can be condensed to just "First time: (...?)" and then separately "(You lay down.)", if that makes sense
  4. re line 1191, "Isabeau: something?", capitalization isn't right
  5. re line 1580, "Isabeau: I'm here, is what I mean! And I love myself so much more now that I did before!" typo, that instead of than
- snack break 1
  1. the text on the following lines should be big (just `class="big"`): 143, 167, 1767
  2. missing a line from isa at line 431 - expression is `NO!_1` and text is `<span class="big">BOOKS?!??!</span>`
  3. at line 513, maybe add a note that it sets variable 357 to 1
  4. at line 560, are we sure it's second time only? i know it does `◆If：Self Var_EventCheck_5 = 2` but it doesn't *seem* right to me
  5. re line 875: ooh the strikethrough doesn't look great there. should probably make it look better with css (not asking anything to be done now, i can handle this at some point, unless you want to!)
  6. re lines 901, 1258, and 1383: for situations like this, we usually do a line break before the line, rather than a space at the start of the line
  7. re lines 1700, 2415, 2548, 2577, 2836: should be in their own `<ul class="dialogue-options">` elements with the 
  8. lines 2989 and 3012 should specify "if you don't know"
  9. line 3059 should either come before 3057 and 3058, or lines 3057 and 3058 should specify "if you don't know"
  10. line 3187 should be a dialogue-note. also there's a typo
  11. lines 3257-3285 can be condensed, even though the game code doesn't do it this way

i am stopping here because that took wAY longer than i thought it would what. hopefully i'll continue this later

**Comment by candycoatedrox** [comment]
setting it to a draft is totally fair lol. honestly the main reason i pr'd now is that i wanted to send this image to feli and i thought it'd be funnier with the snack time pages
all snack pages reformatted

- yeah exactly. didn't want to put the ENTIRE snack time dialogue there when 75% of it was totally irrelevant yknow. will wait for feli input before changing it
- sure, i've been running around reformatting pages, what's like 5 minor things to fix in another page, idc. should be good now
- as i've been reformatting pages i've purely been focusing on reformatting the lines that are already there so i haven't been checking them up against the game or fixing typos/weird phrasing in notes unless something jumps out at me... i should probably be double-checking more. oops. but obviously i'll fix stuff now, lol. couple of notes though:

**iii.** there is one already i think? unless it's not showing up for you for some reason?
image
**iv.** i... honestly can't remember if i've seen this one in game? i only did one friendquest run in my first (and currently only, since i haven't gotten to act 3 yet of my new playthrough) playthrough... the code certainly seems like it is second time only. plus i guess it would make some kind of sense since in your first time through the third snack break with friendquests siffrin doesn't seem aware that odile didn't say "allies at best" that time... he's only noticing it the second time, i guess, because they're listening for it this time? that's how i interpreted it anyway
**v.** honestly i am fully functioning off of using the other pages as examples + an html/css class i took when i was 8, and i remember NOTHING about css. so i unfortunately cannot help with this.
**vii.** i think you forgot to finish writing this one (i get it lol) and i'm confused? i thought `<ul class="dialogue-option">` was only for variations of actual dialogue options, and if the only difference was siffrin portraits you use a regular `<ul>` with the dialogue in a `<li class="option">`?

okay that should be all for now. will check to see the rest of your comments tomorrow!

**Comment by goldstargloww** [comment]
the image is great i enjoy that

> there is one already i think? unless it's not showing up for you for some reason?

yeah that's the thing i was talking about! i more meant like, saying Specifically what variable it changes and what it changes it to. like, easy explanation vs technical explanation. that wasn't too clear, sorry

<br>

> i... honestly can't remember if i've seen this one in game? i only did one friendquest run in my first (and currently only, since i haven't gotten to act 3 yet of my new playthrough) playthrough... the code certainly seems like it is second time only. plus i guess it would make some kind of sense since in your first time through the third snack break with friendquests siffrin doesn't seem aware that odile didn't say "allies at best" that time... he's only noticing it the second time, i guess, because they're listening for it this time? that's how i interpreted it anyway

yeah i have no idea either?? that explanation makes sense and i also thought of that. might need to look through a walkthrough or something on youtube to check? again it definitely seems to line up with the code, it just feels like a little bit of strange decision for it to have been purposefully done that way

<br>

> honestly i am fully functioning off of using the other pages as examples + an html/css class i took when i was 8, and i remember NOTHING about css. so i unfortunately cannot help with this.

fair, no worries, i'll probably get to that at some point myself

<br>

> i think you forgot to finish writing this one (i get it lol) and i'm confused? i thought `<ul class="dialogue-option">` was only for variations of actual dialogue options, and if the only difference was siffrin portraits you use a regular `<ul>` with the dialogue in a `<li class="option">`?

shrug, maybe feli does it differently, or maybe the references you were using are out of date? to be honest i don't really look at how feli does things and i just try to do my own system, and then feli usually cleans it up to be more how he likes it later, which i'm fine with. but i also came up with a lot of the css and formatting, so i designed it with my intended methods in mind. but at the same time i'm also weird because i noticed the wish text in game the very first time it appeared, which according to [the tumblr poll](https://www.tumblr.com/isat-script-project/754351216821141504/example-1-visibility-shadow-effect) doesn't seem to be that common, so maybe i'm just better at parsing and noticing certain things?? i've noticed feli tends to make a lot more visual distinctions in places where i would do it more subtly and still read it perfectly fine if not better, so that's something

either way, this is how i would do it:

```html
<ul class="dialogue-option">
  <li>
    <div class="dialogue-option-only">
      option text goes here
    </div>
  </li>
</ul>
```

and you can add the `indented` class to the `<ul>` to make it look better if needed

it mainly just looks weird to me, having it next to a regular bullet point. shrug

i should sleep it's 3am

**Comment by candycoatedrox** [comment]
OH SO _THAT'S_ WHAT DIALOGUE-OPTION-ONLY DOES. it kept popping up as a suggestion whenever i went to type in a class and i had no idea what it was for. that looks so much better, you're right, i will immediately be changing to that lmao

also the `<div>` isn't necessary i think, you can just put the class right on the `<li>`. or at least if there is a difference it isn't showing up for me.
image

**Comment by goldstargloww** [comment]
checking the css, yeah it doesn't look like you need the div actually, you're correct ^^
and yeah it is a bit unintuitively named, i can go add a comment in the css to clarify

**Comment by goldstargloww** [comment]
re: the "does it actually happen on the second time, that seems weird" thing, replaying it now and yes it does happen on the second time

File: public/overview/acts.html [hunk 1]
```diff
@@ -45,6 +45,7 @@ <h1>ACT 3 & 4</h1>
     <li><a href="/acts/act3-4/intro3.html">ACT 3 Intro</a></li>
     <li>Friendquests
         <ul>
+            <li><a href="/acts/act3-4/bonniequest.html">Bonnie (WIP)</a></li>
```
Comment (goldstargloww) [inline public/overview/acts.html:48]: need to remove (WIP) here?

File: public/acts/act3-4/isaquest.html [hunk 2]
```diff
@@ -2507,7 +2506,7 @@ <h4>ACT 4</h4>
         (You smile back.)
     </p>
 
-    <br>
+    <hr>
 
     <img src="https://instarsandtime.wiki.gg/images/5/55/ISAT_Event_ACT3_Friendquest_Snack_Ending_1.png" alt="The entire group having a picnic. Everyone is happy and smiling while huddled around a variety of food. Bonnie triumphantly holds up a plate of samosas.">
```
Comment (goldstargloww) [inline public/acts/act3-4/isaquest.html:2511]: oh, something i've been doing is adding "image ID:" and "end ID." to the starts and ends respectively of alt text to make it clearer when a screenreader's reading out an image vs normal text. i don't know if any screenreaders announce "this is an image", but windows narrator doesn't at least.

(eg. `image ID: alt text here. end ID.`. "ID" needs to be capitalized to be read out as an acronym)

File: public/rooms/floor3/mirrorroom.html [hunk 3]
```diff
@@ -202,7 +202,7 @@ <h1 id="mirror">Mirror</h1>
                 <summary>ACT 4</summary>
                 
                 <div class="dialogue">
-                    <img src="https://instarsandtime.wiki.gg/images/a/a7/ISAT_Event_ACT4_Mirror_Pic.png">
+                    <img src="https://instarsandtime.wiki.gg/images/a/a7/ISAT_Event_ACT4_Mirror_Pic.png" alt="Image ID: A picture of the party, taken by the mirror. Everyone except Isabeau and Siffrin looks surprised. Siffrin grins manically. End ID.">
```
Comment (goldstargloww) [inline public/rooms/floor3/mirrorroom.html:205]: i'd probably say "siffrin's smile is forced" instead of "grins manically" ^^

**Comment by goldstargloww** [comment]
gonna talk about little things i did in my last commit
1. changed spaces that were there to indicate pauses into `&ensp;`s, just to at least distinguish between them in code
2. there's different commands in rpg maker for fading the screen in and out (`Fadeout Screen`/`Fadein Screen` vs tinting the screen `Tint Screen：(-255,-255,-255,0), X frame(s)` and i also made a distinction for that because they *do* look slightly different. not that important but yknow
3. also i removed/added some `<br>`s for more timing things. for `Wait` commands i generally do one `<br>` per 60 frames (1 second), and then also a `<br>` when the previous message ends in `\m[wait]` or when the next message starts in `\m[wait]`

and then other things
1. you missed a scene in the boulangerie that i added in (was pretty easy to miss dw)
2. also, odile didn't turn to the left, that's just what the code said. the way rpg maker's sprites work is that you pick a spot in an image, and then there's left/right/up/down directions within that section. so this was a special sprite where the directions were actually just variations of the same sprite. so what it Really did was odile looked down a little bit

**Review by felikatze — APPROVED** [review]
Okayo, the usual notes:

For isaquest: Why'd you put "first time only" lines in an ul with a single li? Edge case, I know, but I prefer using the h4 and hr combo for those. It's pretty whatever though. The indentation from the ul also works to separate it i guess, so I'm more ambivalent. It's not _bad_, I just prefer the hr in this case (Feli finish the variations wiki page challenge)

On dialogue-option-only: oh so Gold and i separately made an exact same class for the exact same purpose, lol. Gonna axe class option from the css then, since dialogue-option-only is probably better from the standpoint of "i didn't make it". Does the class need to be on _both_ the ul and the li though? Shouldn't just on the li be enough? Though a redundant class isn't harmful, I suppose.

WAIT WAIT SIDEBAR. my li class option has indentation whereas gold's doesnt. it looks better with the indentation i think. okay yeah we need to discuss this and then change it to be consistent. whichever name the class has i dont care personally it just gotta be indented so as to like. actually fit into the ul. which is what it's supposed to do, yes.

On the reformatted snack break pages: it'd be better to put the act5 text at the very end of the page, so as to not break the flow of the scene. unlike regular room pages with separate objects, the snack breaks are really meant to be seen as one whole thing, yah? additionally, the links to separate sections for the snacks are holdovers from before i knew the details tag exists, so it'd be better to use that instead.

also you may be overusing the details tag for things that aren't dialogue options. i Like It Very Much in snackbreak1 for the odile coinflips but in snackbreak2 you use it for bonnie's fq only snacks into except you DIDNT put the non fq snack intro next to it, which is weird. Probably because you put the act5 inbetween them which i already talked about. i also like it on variations of pictures i know you talked about this with the mirror room before.

I'm gonna approve  and merge this pull requests but I'm gonna take a crack at it before I make a tumblr announcement yah. You also missed like one br in the act5 fqs but I can just do that dw abt it.  yahoo.

**Comment by goldstargloww** [comment]
i just woke up and am in bed on my phone so i'm probably missing some things but

my dialogue option class isn't indented on purpose so you can have it optionally, and use the indented class to make it indented (ie. `class="dialogue-option-only indented"`)

**Pull request merged**